docs/troubleshooting-guide/collection-permissions.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Troubleshooting collection permissions
+---
+
+# Troubleshooting collection permissions
+
+Use this guide if you're having trouble with permissions to _view or edit_ questions or dashboards.
+
+If someone has the wrong level of access to the tables, rows, or columns that are displayed in a question or dashboard, see [Troubleshooting data permissions][troubleshooting-data-permissions].
+
+## People can't access a dashboard from a collection that they have permissions for
+
+1. Check where each _question_ on the dashboard is saved (i.e., which collection).
+2. If the saved question is in a collection that people don't have access to, you can either:
+    - Grant people permission to the collection.
+    - Move the saved question to a collection with broader permissions.
+
+**Explanation**
+
+Moving a dashboard to a collection with permissions does not automatically move all of the saved questions into the same collection. If the saved questions are stored in a restrictive collection (such as someone's personal folder), then other people will not be able to view those cards.
```

Comment:
```suggestion
Moving a dashboard to a different collection doesn't move the dashboard's questions into that collection. If the saved questions are stored in a restricted collection (such as someone's personal folder), then other groups won't be able to view those cards. You'll need to move those questions into a collection that your target groups have permissions for.
```
We should be mindful/consistent that permissions are applied to groups, not people.